src/transformers/models/roc_bert/modeling_roc_bert.py

```diff
@@ -52,6 +52,22 @@
 _CONFIG_FOR_DOC = "RoCBertConfig"
 _TOKENIZER_FOR_DOC = "RoCBertTokenizer"
 
+# Base model docstring
+_EXPECTED_OUTPUT_SHAPE = [1, 8, 768]
+
+# Token Classification output
+_TOK_CLASS_EXPECTED_OUTPUT = ['LABEL_0', 'LABEL_1', 'LABEL_0', 'LABEL_0', 'LABEL_0', 'LABEL_1', 'LABEL_1', 'LABEL_0', 'LABEL_1', 'LABEL_1', 'LABEL_1', 'LABEL_1', 'LABEL_1', 'LABEL_1']
+
+# SequenceClassification docstring
+_SEQ_CLASS_EXPECTED_LOSS = 0.0
+_SEQ_CLASS_EXPECTED_OUTPUT = "'LABEL_1'"
```

Comment:
Don't we have any model with labels? Adding those examples with abstract label just does not make any sense...

Reply:
Would be better no?